pkg/cloud/client/environments.go

```diff
@@ -0,0 +1,35 @@
+package client
+
+import (
+	"github.com/kubeshop/testkube/pkg/http"
+)
+
+func NewEnvironmentsClient(rootDomain, token string) *EnvironmentsClient {
+	return &EnvironmentsClient{
+		RESTClient: RESTClient[Environment]{
+			BaseUrl: "https://api." + rootDomain,
+			Path:    "/environments",
```

Comment:
why is third level domain `api` hardcoded?

Reply:
as it'll be always api everytwhere - we have 3 possible clusters all are based on converntions now, even helm charts are built that way. It simplifies how stuff is built.